CheerUp/urls.py

```diff
@@ -21,4 +21,6 @@
     path("admin/", admin.site.urls),
     path("events/", include("events.urls")),
     path("locations/", include("location.urls")),
+    path("accounts/", include("accounts.urls")),
+    path("accounts/", include("django.contrib.auth.urls")),
```

Comment:
2 includes for same paths
Is it correct?

Reply:
Django may use the URL from the first included set (accounts.urls in this case) and ignore the second. Added intentionally. :)